scenarios/kubernetes_bazel.py

```diff
@@ -39,21 +39,28 @@ def check_output(*cmd):
 
 
 class Bazel(object):
-    def __init__(self, batch):
+    def __init__(self, batch, *cfgs):
         self.batch = batch
+        self.cfgs = cfgs
 
-    def check(self, *cmd):
-        """wrapper for check('bazel', *cmd) that respects batch"""
+    def _commands(self, cmd, *args, **kw):
+        commands = ['bazel']
         if self.batch:
-            check('bazel', '--batch', *cmd)
-        else:
-            check('bazel', *cmd)
+            commands.append('--batch')
```

Comment:
--batch has been deprecated for a while (~one year, 0.15.0), we could probably get rid of this.

https://github.com/bazelbuild/bazel/commit/2d53deee92ed6134e861ab6d0a8a0355510bb57d

Reply:
Done